label-application-rule-config-service-impl/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+plugins {
+    `java-library`
+    jacoco
+    id("org.hypertrace.jacoco-report-plugin")
+}
+
+dependencies {
+    api(projects.labelApplicationRuleConfigServiceApi)
+    implementation(projects.configServiceApi)
+    implementation(projects.configProtoConverter)
+    implementation(libs.typesafe.config)
+    implementation(libs.protobuf.javautil)
+    implementation(libs.slf4j.api)
+    implementation(libs.guava)
+    implementation(libs.rxjava3)
```

Comment:
please go through the dependencies and limit to what is needed - I checked a couple (rxjava and guava) and neither is used.